csrc/host_ir/executor.cpp

```diff
@@ -149,6 +149,22 @@ void HostIrExecutor::handle(Wait* wait) {
   works_.erase(communication);
 }
 
+void HostIrExecutor::handle(kir::ForLoop* for_loop) {
+  NVF_ERROR(for_loop->start()->isConstInt());
+  NVF_ERROR(for_loop->step()->isConstInt());
+  NVF_ERROR(for_loop->stop()->isConstInt());
+  auto start = for_loop->start()->value().as<int64_t>();
+  auto step = for_loop->step()->value().as<int64_t>();
+  auto stop = for_loop->stop()->value().as<int64_t>();
+
+  for (auto i = start; i < stop; i += step) {
+    for (auto j : c10::irange(for_loop->body().size())) {
```

Comment:
```suggestion
    for (Expr* e : for_loop->body().exprs()) {
```

Reply:
Because `j` is only used to index expressions, the above is slightly simpler.

Reply:
right! I fixed it